extensions/visual-studio-code-snippet-manager/src/components/SearchSection.tsx

```diff
@@ -0,0 +1,99 @@
+import { Action, ActionPanel, Alert, confirmAlert, List, popToRoot, useNavigation } from "@raycast/api";
+import Editor from "./Editor";
+import Search from "./Search";
+import fs from "fs";
+import { vsCodeInsidersPath, vsCodePath } from "../data";
+type Props = {
+  items: Required<Item>[] | null;
+  title: "vscode" | "vscode-insiders";
+};
+const SearchSection = ({ items, title }: Props) => {
+  const { push } = useNavigation();
+  const deleteSnippet = (key: string) => {
+    let path = "";
+    if (title === "vscode") {
+      path = vsCodePath;
+    } else {
+      path = vsCodeInsidersPath;
+    }
+    const obj = JSON.parse(fs.readFileSync(path).toString()) as {
+      [key: string]: { prefix: string; description: string; title: string; body: string[]; scope: string };
+    };
+    delete obj[key];
+    fs.writeFileSync(path, JSON.stringify(obj));
+  };
+  return (
+    <List.Section title={title}>
+      {items?.map((v) => {
+        return (
+          <List.Item
+            keywords={[...v.scope, v.body, v.prefix, v.description]}
+            key={v.id}
+            title={v.id}
+            subtitle={v.description}
+            actions={
```

Comment:
Could you update the actions so that they stick to the [guidelines](https://developers.raycast.com/basics/prepare-an-extension-for-store#action-panel)? 😸 
That is:
- actions should be `Title Case`
- all actions should have icons, or none of them should